x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/data/BlockBuilderCopyFromTests.java

```diff
@@ -36,7 +36,7 @@ public static List<Object[]> params() {
                 continue;
             }
             for (boolean nullAllowed : new boolean[] { false, true }) {
-                if (e == ElementType.AGGREGATE_METRIC_DOUBLE) {
+                if (e == ElementType.AGGREGATE_METRIC_DOUBLE || e == ElementType.LONG_RANGE) {
```

Comment:
I think elasticsearch does allow multi-values for date-range. Is this limit here because we have not yet implemented this support?